Automattic/woocommerce-payments
# Fix payment method section missing for Affirm and Afterpay on transaction details page

Fixes #7343 

#### Changes proposed in this Pull Request


This pull request adds the payment method section to the details page of a transaction paid with Affirm or Afterpay.






### Affirm

**Before**

<img width="1154" alt="Screenshot 2023-10-04 at 16 25 24" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/b10ea3c2-b568-4adc-8756-713ca9fb0630">

**After**

<img width="1154" alt="Screenshot 2023-10-04 at 16 22 46" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/eb0c0753-2d59-4d2d-aaf5-ebdb9bea7344">


### Afterpay

**Before**

<img width="1154" alt="Screenshot 2023-10-04 at 16 26 33" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/6d6b9a0c-cec9-4a4f-bfdc-94b2483014b3">

**After**

<img width="1154" alt="Screenshot 2023-10-04 at 16 25 51" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/e554f8d4-510a-4b20-b253-aff06ea06d6f">

#### Testing instructions





* You'll need at least one Affirm and Afterpay transactio.
* Go to WP Admin > Payments > Transactions
* Locate one Affirm transaction and click it to go the details page
* Scroll to bottom and check that the payment method section appears.
* Repeat the process with one Afterpay transaction
-------------------


**Post merge**
